music_assistant/server/providers/plex/__init__.py

```diff
@@ -413,10 +492,14 @@ async def _parse_album(self, plex_album: PlexAlbum) -> Album:
                     item_id=str(album_id),
                     provider_domain=self.domain,
                     provider_instance=self.instance_id,
-                    url=plex_album.getWebURL(),
+                    url=plex_album.getWebURL(self._baseurl),
                 )
             },
         )
+        # Only add 5-star rated albums to Favorites. rating will be 10.0 for those.
+        # TODO: Let user set threshold?
+        # album.favorite = True if plex_album._data.attrib['rating'] == "10.0" else False
+
```

Comment:
This is outdated ? Or another reason why its commented out ?

Reply:
Oh yeah. I hadn't actually rated any of my albums in Plex, only tracks. Suppose I can mark a few and test that.

Reply:
Done.